Shared/Samples/Sketch on the map/README.md

```diff
@@ -0,0 +1,35 @@
+# Sketch on the map
```

Comment:
The common design changed the name to `Sketch on map` without "the". Please rename

- the folder
- README title
- title in metadata
- swift filename
- view type name